### PR TITLE
MINOR: Missing punctuation marks in quickstart

### PR DESCRIPTION
**Pull request by vitojeng**
Minor fix for missing punctuation marks in the quickstart.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: docs/streams/quickstart.html
Comment (vvcephei) [inline docs/streams/quickstart.html:60]: nit: maybe we can reformat this for readability like this:
```
KStream&lt;String, String&gt; textLines = builder.stream(
  "streams-plaintext-input",
  Consumed.with(stringSerde, stringSerde)
);
```

**Review by vvcephei — COMMENTED** [review]
Thanks for this!

Overall, it looks good to me; just one question.

File: docs/streams/quickstart.html
Comment (vitojeng) [reply docs/streams/quickstart.html:60]: Thanks @vvcephei. I'll update later.

**Review by vvcephei — APPROVED** [review]
LGTM! Thanks, @vitojeng

**Comment by vvcephei** [comment]
@bbejeck , do you mind taking at look at this also?

**Review by bbejeck — APPROVED** [review]
Thanks for the contribution @vitojeng! LGTM I'll merge once the fix per @vvcephei's comment is in.

**Comment by bbejeck** [comment]
retest this please

**Comment by vitojeng** [comment]
@bbejeck Thanks for the reminder. @vvcephei 's comment already resolved.

**Comment by vvcephei** [comment]
Confirmed. Sorry for the confusion @bbejeck , the line that I commented on didn't change, so my comment was still visible, but @vitojeng reformatted the code as requested.

**Comment by bbejeck** [comment]
retest this please

**Comment by bbejeck** [comment]
@vitojeng @vvcephei thanks for clearing up.  I'm just waiting for jenkins to run then I'll merge.

**Comment by bbejeck** [comment]
Java 8 and 11 passed, we don't use Java 10 checks anymore, so I'm merging this.

**Comment by bbejeck** [comment]
merged #5755 into trunk

**Comment by vitojeng** [comment]
Thanks @bbejeck and @vvcephei !

**Pull request merged**
